lib/assignment/widgets/assignments_screen.dart

```diff
@@ -231,4 +206,29 @@ class _AssignmentCardState extends State<AssignmentCard> {
         ),
     ];
   }
+
+  Widget _buildCourseChip(BuildContext context) {
+    if (_course != null) {
+      return CourseChip(
+        _course,
+        onPressed: () {
+          // TODO(JonasWanke): filter list by course, https://github.com/schul-cloud/schulcloud-flutter/issues/145
+        },
+      );
+    }
+    return widget.assignment.courseId != null
```

Comment:
`widget.assignment.courseId` cannot be null anymore.
Come to think of it, wouldn't it make sense to put this logic inside the `CourseChip`? It applies to all the chips on all screens and you could just pass an `Id<Course>` instead of referring to it as `widget.assignment.courseId` all over the place.

Reply:
Yep; that's already implemented in #257